js/modules/k6/http/response.go

```diff
@@ -141,3 +144,77 @@ func (res *HTTPResponse) Html(selector ...string) html.Selection {
 	}
 	return sel
 }
+
+func (res *HTTPResponse) SubmitForm(args ...goja.Value) (*HTTPResponse, error) {
+	rt := common.GetRuntime(res.ctx)
+
+	formSelector := "form"
+	submitSelector := "[type=\"submit\"]"
+	var fields map[string]goja.Value
+	var requestParams goja.Value
```

Comment:
If this line is changed to `requestParams := goja.Null()` we can remove the if/else at the end of the method when calling `Request()`.